webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -9628,6 +9627,27 @@ var userOverride = {
                 return '';
             };
 
+            
+             _this.getMessageCreatedAtTime=function(createdAtTime){
+                 if(TIME_FORMAT_24HOURS){
+                    var msgtime = new Date(createdAtTime);
+                    function addZero(i) {
+                        if (i < 10) {i = "0" + i}
```

Comment:
The variables declared without the var keyword becomes global variables
it should be declared

@Prochnost55 can confirm